DataSetExplorer/DataSetSerializer/SingleAnnotatorSanityCheckExporter.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using CodeModel.CaDETModel.CodeItems;
+using DataSetExplorer.DataSetBuilder.Model;
+using OfficeOpenXml;
+
+namespace DataSetExplorer.DataSetSerializer
+{
+    class SingleAnnotatorSanityCheckExporter
+    {
+        private readonly string _templatePath = "../../../DataSetSerializer/Template/Single_Annotator_Sanity_Check_Template.xlsx";
+        private readonly string _exportPath;
+        private ExcelPackage _excelFile;
+        private ExcelWorksheet _sheet;
+
+        public SingleAnnotatorSanityCheckExporter(string exportPath)
+        {
+            _exportPath = exportPath;
+        }
+
+        public void Export(List<Tuple<DataSetInstance, Dictionary<CaDETMetric, double>>> dataSetInstances,
+            string fileName, int annotatorId)
+        {
+            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
+            _excelFile = new ExcelPackage(new FileInfo(_templatePath));
+            _sheet = _excelFile.Workbook.Worksheets[0];
+            PopulateTemplate(dataSetInstances, annotatorId);
+            Serialize(fileName + annotatorId);
+        }
+
+        private void PopulateTemplate(List<Tuple<DataSetInstance, Dictionary<CaDETMetric, double>>> instances, int annotatorId)
```

Comment:
This tuple might be a new domain concept. When generating a dataset we might want to include these metrics, so we should think about expanding our domain model.